apps/showcase/src/components/utilities/date-picker-input/date-picker-input-pres.config.ts

```diff
@@ -0,0 +1,8 @@
+import type {Configuration} from '@o3r/core';
+import {computeItemIdentifier} from '@o3r/core';
+
+export interface DatePickerInputPresConfig extends Configuration {}
```

Comment:
Is config really needed ?

Reply:
same as previous comment. to be checked

Reply:
I understand why you must have done that 
but I don't understand why `T` has to extends `DynamicConfigurable`
as configuration for `presKey` is the one of the parent component

Reply:
As params you give the default presenter, which is a component having context and configuration, and a key used to search in presenters map, a custom presenter which has to expend the Type<T>.
Why we force the Context and DynamicConfigurable<Configuration> is because initially we generated all our otter components with context, config, loc etc. Nowadays we don;t force this in the generators, so we might revisit the c11n mechanism to accept more generic components too.

Reply:
For the context it could make sense as we are binding inputs and outputs

Reply:
Yes. I have created an issue to track this
https://github.com/AmadeusITGroup/otter/issues/1197